### ISSUE TITLE
metadata.json 

### ISSUE THREAD

**Issue by PradipH31** [lap00zza/9anime-Companion#79]
**Select your issue type:** (check at least one)
- [ ] Bug
- [ ] Question
- [ x] Suggestion
- [ ] Other (please describe):

**Describe your issue:**
How about adding facility of creating metadata.json while downloading from G1 server?

**Comment by lap00zza** [comment]
Its possible, but this problem is already solved by downloading via browser. @WEBhicham what do you think? Info about metadata.json: https://github.com/lap00zza/Grabber#what-is-metadatajson

**Comment by PradipH31** [comment]
If I had to download Detective Conan for example it would be way difficult with the browser with 800+ episodes.

**Comment by lap00zza** [comment]
True. And since G1 sends direct media links renaming them via title param is not possible. Alright, I will accept a PR for this (or, implement it myself sometime next week).

**Comment by PradipH31** [comment]
If that can not be then I would like to request metadata.json sample for anime series. I tried to create one using the example from One Piece Gold movie but for series, it only renamed the last episode.

**Comment by lap00zza** [comment]
The basic format is this
```json
{
	"animeName": "NAME",
	"animeUrl": "9anime_Episode_URL",
	"files": [{
			"original": "ORIGINAL_NAME.ext",
			"real": "PROPER_NAME.ext"
	}, {
			"original": "bVTiwZTHZS2Lmme.mp4",
			"real": "one piece film_ gold-ep_001-standard.mp4"
	}],
	"timestamp": "TIMESTAMP",
	"server": "SERVER"
}
```

**Comment by PradipH31** [comment]
Thank you.
